video/image_writer.c

```diff
@@ -262,7 +297,237 @@ static bool write_jpeg(struct image_writer_ctx *ctx, mp_image_t *image, FILE *fp
 
     jpeg_destroy_compress(&cinfo);
 
-    return true;
+    return !fclose(fp);
+}
+
+#endif
+
+#if HAVE_AVIF_MUXER
+
+static void log_side_data(struct image_writer_ctx *ctx, AVPacketSideData *data,
+                          int size)
+{
+    static char dbgbuff[129];
```

Comment:
Why is this static?

Reply:
I meant dbgbuff. The function can be static, but why is the debug buffer static?